packages/vector_graphics/lib/src/vector_graphics.dart

```diff
@@ -674,7 +686,7 @@ class VectorGraphicUtilities {
   ///
   /// It is the caller's responsibility to handle disposing the picture when
   /// they are done with it.
-  Future<PictureInfo> loadPicture(
+  Future<PictureInfo?> loadPicture(
```

Comment:
This looks like a breaking change, so will need to be evaluated in the context of https://github.com/flutter/flutter/issues/157626

If the decision there is (as I hope) to start using semver for `vector_graphics*`, this would need to be versioned accordingly, or would need a non-breaking alternative.

Reply:
Rather than making it null, we can throw/catch an exception I think.